lib/Importer/ProtobufLoader.cpp

```diff
@@ -34,6 +34,10 @@ Tensor *ProtobufLoader::getTensorByName(llvm::StringRef name) {
   return tensors_[name];
```

Comment:
Maybe not critical, but now `tensors_` can contain nulls instead of valid Tensor pointers. So we'd need to add assertion to this method: not only check that `tensors_.count(name)`, but also `tensors_[name]`.

Reply:
Ew.  Yeah, I don't think I want [] here.

Reply:
Great catch, btw!